replica/database.cc

```diff
@@ -2031,6 +2031,31 @@ future<> database::flush(const sstring& ksname, const sstring& cfname) {
     return cf.flush();
 }
 
+future<> database::flush_on_all(utils::UUID id) {
+    return container().invoke_on_all([id] (replica::database& db) {
+        return db.find_column_family(id).flush();
+    });
+}
+
+future<> database::flush_on_all(std::string_view ks_name, std::string_view table_name) {
+    return flush_on_all(find_uuid(ks_name, table_name));
```

Comment:
Because `find_column_family(ks, cf)` is per replica::database shard instance and returns a shard-local column_family&
the uuid returned by find_uuid(ks, cf) can be used on any shard to locate the table more efficiently
than looking up the coloumn family by (ks_name, cf_name) on every shard..